guides/source/action_controller_overview.md

```diff
@@ -143,15 +141,15 @@ So for example, if you are sending this JSON content:
 { "company": { "name": "acme", "address": "123 Carrot Street" } }
 ```
 
-You'll get `params[:company]` as `{ "name" => "acme", "address" => "123 Carrot Street" }`.
+Your controller will receive `params[:company]` as `{ "name" => "acme", "address" => "123 Carrot Street" }`.
 
-Also, if you've turned on `config.wrap_parameters` in your initializer or calling `wrap_parameters` in your controller, you can safely omit the root element in the JSON parameter. The parameters will be cloned and wrapped in the key according to your controller's name by default. So the above parameter can be written as:
+Also, if you've turned on `config.wrap_parameters` in your initializer or called `wrap_parameters` in your controller, you can safely omit the root element in the JSON parameter. In this case, by default, the parameters will be cloned and wrapped with a key chosen based on your controller's name. So the above JSON POST can be written as:
```

Comment:
Wouldn't say "by default," since you're explicitly overriding the default behavior.